modules/playing-card/widgets/playing-card-widget.php

```diff
@@ -0,0 +1,274 @@
+				'type' => Controls_Manager::SELECT,
+				'default' => '❤️',
+				'options' => $suite_options,
+			]
+		);
+
+		$repeater->end_controls_tab();
+
+		$repeater->start_controls_tab(
+			'style',
+			[
+				'label' => esc_html__( 'Style', 'elementor' ),
+			]
+		);
+
+		$repeater->add_responsive_control(
+			'card_padding',
+			[
+				'label' => esc_html__( 'Padding', 'elementor' ),
+				'type' => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} {{CURRENT_ITEM}}' => 'padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				],
+			]
+		);
+
+		$repeater->end_controls_tab();
+
+		$this->add_control(
+			'playing_cards_list',
+			[
+				'label' => esc_html__( 'Playing cards', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [
+					[
+						'playing_card' => [
+							'card_number' => 'A',
+							'card_suit' => '❤️',
+						],
+					],
+				],
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	private function add_playing_cards_style_section() {
+		$this->start_controls_section(
+			'section_playing_card_style',
+			[
+				'label' => esc_html__( 'Playing Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_responsive_control(
+			'playing_card_item_space_between',
+			[
+				'label' => esc_html__( 'Space between Items', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 200,
+					],
+				],
+				'default' => [
+					'size' => 0,
+				],
+				'selectors' => [
+					'{{WRAPPER}} > .elementor-widget-container > .e-playing-cards-wrapper' => 'gap: {{SIZE}}{{UNIT}}',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	/**
+	 * Render playing card widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render() {
+		$settings = $this->get_settings_for_display();
+
+		?>
+		<div class="e-playing-cards-wrapper">
+			<?php
+			foreach ( $settings['playing_cards_list'] as $index => $item ) {
+				$card_id = 'card_' . $index;
+
+				$this->add_render_attribute( $card_id, 'class', [
+					'e-playing-cards-wrapper-item',
+					'elementor-repeater-item-' . $item['_id'],
+				]);
+
+				$card_number = $item['card_number'];
+				$card_suit   = $item['card_suit'];
+				$card_color  = $this->suites_map[ $card_suit ]['color'];
+				?>
+				<div <?php $this->print_render_attribute_string( $card_id ); ?>>
+					<div class="e-playing-cards-wrapper-item-top"><?php echo esc_attr( $card_suit ); ?></div>
+					<div class="e-playing-cards-wrapper-item-center e-playing-cards-wrapper-item--number" style="color: <?php echo esc_html( $card_color ); ?>"><?php echo esc_attr( $card_number ); ?></div>
```

Comment:
```suggestion
					<div class="e-playing-cards-item-center e-playing-cards-item--number" style="color: <?php echo esc_html( $card_color ); ?>"><?php echo esc_attr( $card_number ); ?></div>
```

Reply:
Try to avoid the use of inline style unless there is no other option.
A better choice would be to create the following classes: `.red`, `.black` in the CSS file and add them using the `add_render_attribute` function.